.gitignore

```diff
@@ -10,4 +10,8 @@ test/js/dest/
 *~
 .livereload
 *.orig
+*.css.map
+*.js.map
 dist/
+docs/assets/
+docs/search-data.json
```

Comment:
I decided to leave the HTML files in `docs/` checked into the repo, because I often use the diffs of the compiled HTML as a form of functional testing.